Add the scattershot boson sampling example in the gallery

artix41 opened the pull request:
**Description of the Change:**

Add the scattershot boson sampling example in the gallery

**Benefits:**

New example in the gallery

**Possible Drawbacks:**

**Related GitHub Issues:**

josh146 commented:
Fantastic contribution, thank you Arthur. I've made a small commit adding your notebook to the gallery page in the documentation - check it out and make sure everything renders as it should (you can compile the docs by running `make docs` in the top-level directory, and then opening the file `docs/_build/html/index.html` in your browser.

I've also done some light editing of the notebook to make sure that everything renders correctly when embedded within our documentation. In particular, I've added your byline to the notebook (double check that this is written how you'd like it!) and also modified the in-text citations to avoid using square brackets. Digits in square brackets (i.e. `[1]`) unfortunately clash with our documentation reference page.

Once you've double checked the changes, I'll merge it in.

artix41 commented:
Looks good! I've just saved the notebook with another sampling at the end
for a better visualization.

It's ready for the merge I think!

On 5 August 2018 at 18:12, Josh Izaac <notifications@github.com> wrote:

> Fantastic contribution, thank you Arthur. I've made a small commit adding
> your notebook to the gallery page in the documentation - check it out and
> make sure everything renders as it should (you can compile the docs by
> running make docs in the top-level directory, and then opening the file
> docs/_build/html/index.html in your browser.
>
> I've also done some light editing of the notebook to make sure that
> everything renders correctly when embedded within our documentation. In
> particular, I've added your byline to the notebook (double check that this
> is written how you'd like it!) and also modified the in-text citations to
> avoid using square brackets. Digits in square brackets (i.e. [1])
> unfortunately clash with our documentation reference page.
>
> Once you've double checked the changes, I'll merge it in.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/XanaduAI/strawberryfields/pull/30#issuecomment-410530462>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABGrUCZMj62GbsahrnqDOrRL14m-WtoTks5uNxlwgaJpZM4VvZ23>
> .
>

artix41 commented:
Looks good! I've just saved the notebook with another sampling at the end for a better visualization. 

It's ready for the merge I think!
